tests/unit/OrgPerson.spec.ts

```diff
@@ -396,16 +445,21 @@ describe('Org/Person component', () => {
   it('Shows popup if there is an existing completing party', async () => {
     const wrapper = createComponent(validIncorporator, NaN, 0, validPersonData)
 
+    // verify that popup is not yet displayed
+    expect(wrapper.find('.confirm-dialog').exists()).toBe(false)
+
+    // check the Completing Party box
     const checkbox = wrapper.find(completingPartyChkBoxSelector)
     checkbox.setChecked(true)
     await Vue.nextTick()
 
-    expect(wrapper.vm.$refs.reassignCpDialog).toBeTruthy()
```

Comment:
This always return truthy, even if the dialog is not displayed.
My new code checks whether the dialog div is present in the DOM.